src/disp_s1/validate.py

```diff
@@ -358,17 +400,16 @@ def rewrap(phi: np.ndarray) -> np.ndarray:
     logger.info(f"Mean absolute re-wrapped phase error: {mean_abs_err:.5f} rad")
     logger.info(f"Max absolute re-wrapped phase error: {max_abs_err:.5f} rad")
 
-    atol_radians = atol * 4 * np.pi / wavelength
-    noncongruent_count = np.sum(abs_wrapped_diff > atol_radians)
+    noncongruent_count = np.sum(abs_wrapped_diff > atol)
```

Comment:
We should just double check that I didn't compress too much from mantissa rounding so that we need a slightly lower tolerance. I think it should be fine though